Assets/BowAndArrowImproved/Scripts/UpdateScoreOnDeath.cs

```diff
@@ -13,8 +13,8 @@ public void Start()
         }
     }
 
-    public void UpdateScoreboard()
+    public void UpdateScoreboard(GameObject arg0)
```

Comment:
why do you need the extra argument here?

Reply:
must use it so I can add a listener to this method from code, changed the name to enemyToAddListener

Reply:
needed it so I could add a listener to the relevant enemy game object from the code
**changed implementation and it's been deleted.**